packages/asset-service/sample.env

```diff
@@ -0,0 +1 @@
+REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=https://dev-api.ethereum.shapeshift.com
```

Comment:
can you add a line or two of docs to this asset service readme to copy this to `.env` before running generate?

Reply:
Good call, done.